### PR TITLE
Documentation improvements for RC.1

### PR DESCRIPTION
**Pull request by lukekim**

### CODE REVIEW

**Review by Sevenannn — DISMISSED** [review]

File: spiceaidocs/docs/api/jdbc/index.md [hunk 1]
```diff
@@ -16,17 +16,14 @@ Spice supports JDBC clients through a JDBC driver implementation based on the [F
...
 
 ### Add the driver to your application
 
 Follow the instructions specific to your application for adding a custom JDBC driver. Examples:
 
-**Tableau**:
-    - Windows: `C:\Program Files\Tableau\Drivers`
-    - Mac: `~/Library/Tableau/Drivers`
-    - Linux: `/opt/tableau/tableau_driver/jdbc` - Start or restart Tableau
+**Tableau**: - Windows: `C:\Program Files\Tableau\Drivers` - Mac: `~/Library/Tableau/Drivers` - Linux: `/opt/tableau/tableau_driver/jdbc` - Start or restart Tableau
```
Comment (phillipleblanc) [inline spiceaidocs/docs/api/jdbc/index.md]: Looks like this formatting got broken

File: spiceaidocs/docs/clients/grafana/index.md [hunk 2]
```diff
@@ -39,65 +39,64 @@ global:
 
 ## Local Quickstart
 
-This tutorial creates and configures Grafana and Prometheus locally to scrape and display metrics from several Spice instances. It assumes:
-    - Two Spice runtimes, `spiced-main` and `spiced-edge`, are running on `127.0.0.1:9091` and `127.0.0.1:9092` respectively.
+This tutorial creates and configures Grafana and Prometheus locally to scrape and display metrics from several Spice instances. It assumes: - Two Spice runtimes, `spiced-main` and `spiced-edge`, are running on `127.0.0.1:9091` and `127.0.0.1:9092` respectively.
```
Comment (phillipleblanc) [inline spiceaidocs/docs/clients/grafana/index.md]:
```suggestion
This tutorial creates and configures Grafana and Prometheus locally to scrape and display metrics from several Spice instances. It assumes: 
  - Two Spice runtimes, `spiced-main` and `spiced-edge`, are running on `127.0.0.1:9091` and `127.0.0.1:9092` respectively.
```

File: spiceaidocs/docs/components/embeddings/index.md [hunk 3]
```diff
@@ -7,12 +7,12 @@ pagination_prev: null
 pagination_next: null
 ---
 
-Embedding models are used to convert raw text into a numerical representation that can be used by machine learning models.
-
-Spice supports running embedding models locally, or use remote services such as OpenAI, or [la Plateforme](https://console.mistral.ai/).
+Embedding models convert raw text into numerical representations that can be used by machine learning models. Spice supports running embedding models locally or using remote services such as OpenAI or [la Plateforme](https://console.mistral.ai/).
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/embeddings/index.md:10]: Is `la Plateforme` correct?

File: spiceaidocs/docs/intelligent-applications/index.md [hunk 4]
```diff
@@ -7,14 +7,50 @@ pagination_prev: null
...
+
+Applications interact with the Spice runtime through high-performance APIs, calling machine learning models for inference tasks such as predictions, recommendations, or anomaly detection. These models are colocated with the runtime, allowing them to leverage the same locally materialized datasets. For example, an e-commerce application could use this infrastructure to provide real-time product recommendations based on user behavior, or a manufacturing system could detect equipment failures before they happen by analyzing time-series sensor data.
+
+As the application runs, contextual and environmental data—such as user actions or external sensor readings—is ingested into the runtime. This data is replicated back to centralized compute clusters where machine learning models are retrained and fine-tuned to improve accuracy and performance. The updated models are automatically versioned and deployed to the runtime, where they can be A/B tested in real time. This continuous feedback loop ensures that applications evolve and improve without manual intervention, reducing time to value while maintaining model relevance.
```
Comment (phillipleblanc) [inline spiceaidocs/docs/intelligent-applications/index.md]:
```suggestion
As the application runs, contextual and environmental data—such as user actions or external sensor readings are ingested into the runtime. This data is replicated back to centralized compute clusters where machine learning models are retrained and fine-tuned to improve accuracy and performance. The updated models are automatically versioned and deployed to the runtime, where they can be A/B tested in real time. This continuous feedback loop ensures that applications evolve and improve without manual intervention, reducing time to value while maintaining model relevance.
```

**Review by phillipleblanc — DISMISSED** [review]

**Review by ewgenius — DISMISSED** [review]

File: spiceaidocs/docs/api/jdbc/index.md (hunk 1 above)
Comment (ewgenius) [inline spiceaidocs/docs/api/jdbc/index.md]:
```suggestion
**Tableau**:

- Windows: `C:\Program Files\Tableau\Drivers`
- Mac: `~/Library/Tableau/Drivers`
- Linux: `/opt/tableau/tableau_driver/jdbc`
- Start or restart Tableau
```

Comment (ewgenius) [reply spiceaidocs/docs/api/jdbc/index.md]: Fixed

File: spiceaidocs/docs/components/embeddings/index.md (hunk 3 above)
Comment (ewgenius) [reply spiceaidocs/docs/components/embeddings/index.md:10]: Yes, it is correct https://mistral.ai/news/la-plateforme/

**Review by ewgenius — APPROVED** [review]

**Pull request merged**